pandas/tests/frame/constructors/test_from_dict.py

```diff
@@ -189,3 +184,16 @@ def test_frame_dict_constructor_empty_series(self):
         # it works!
         DataFrame({"foo": s1, "bar": s2, "baz": s3})
         DataFrame.from_dict({"foo": s1, "baz": s3, "bar": s2})
+
+    def test_from_dict_scalars_requires_index(self):
```

Comment:
Note: I moved this function down here, so that all functions that test exception behaviours are grouped together. Not sure if that complies to the standard, but it certainly seems better

Reply:
Great!